cset-workflow/meta/rose-meta.conf

```diff
@@ -115,697 +232,1353 @@ sort-key=met1
+value-titles=Case Study, Trial
+trigger=template variables=CSET_CASE_DATES: "case_study";
+        template variables=CSET_TRIAL_START_DATE: "trial";
+        template variables=CSET_TRIAL_END_DATE: "trial";
+        template variables=CSET_TRIAL_CYCLE_PERIOD: "trial";
+compulsory=true
+sort-key=a1
+
+# Case study settings.
+[template variables=CSET_CASE_DATES]
+ns=Models and Cases
+title=Case study dates
+description=List of datetimes of cases.
+help=This should be a python list of ISO 8601 datetime strings indicating the
+    forecast initiation time (AKA data time) of the data. E.g. 2000-01-01T00:00Z
+    Ensure that it is consistent with your data’s first validity time. If not
+    then a warning that cubes can not be loaded is raised.
+type=python_list
+compulsory=true
+sort-key=b1
 
-    Standard deletes most intermediate files, save for the final data that is
-    plotted.
-values=0, 1, 2
-value-titles=None, Debug, Standard
+# Trials settings.
+[template variables=CSET_TRIAL_START_DATE]
+ns=Models and Cases
+title=Trial start date
+description=Start date of the trial.
+help=The start date of the trial, in ISO 8601 format. This is the first date
+    that the workflow will run from. For example: 2000-01-01T00:00Z
+type=quoted
 compulsory=true
+sort-key=c1
 
-[template variables=COLORBAR_FILE]
-ns=General
-description=Filepath and name for colorbar details of each variable i.e.
-    name_of_filepath/name_of_filename. An example file is available under
-    CSET/cset-workflow/extra-meta/colorbar_dict_alphabetical.json
-help=
+[template variables=CSET_TRIAL_END_DATE]
+ns=Models and Cases
+title=Trial end date
+description=End date of the trial. If blank, the workflow will run indefinitely.
+help=The end date of the trial, in ISO 8601 format. This is the last date that
+    the workflow will run to. If blank, the workflow will run indefinitely.
+    For example: 2000-01-01T00:00Z
 type=quoted
 compulsory=true
+sort-key=c2
 
-[template variables=PLOT_RESOLUTION]
-ns=General
-description=Resolution of output plot in dpi.
-help=This is passed through to the plotting operators and sets the resolution
-    of the output plots to the given number of pixels per inch. If unset
-    defaults to 100 dpi. The plots are all 8 by 8 inches, so this corresponds
-    to 800 by 800 pixels.
-type=integer
+[template variables=CSET_TRIAL_CYCLE_PERIOD]
+ns=Models and Cases
+title=Trial cycle period
+description=The period between model runs of the trial.
+help=The cycle period of the trial, as an ISO 8601 duration. This is the time
+    between each forecast initiation of the model. For example: PT48H
+type=quoted
 compulsory=true
+sort-key=c3
 
-[template variables=WEB_DIR]
-ns=General
-description=Path to directory that is served by the webserver.
-help=This will probably be under $HOME/public_html or similar. You will want to
-    ensure a subfolder is used as multiple files will be written here.
+[template variables=CSET_ANALYSIS_PERIOD]
```

Comment:
glad to see this. This answers an earier question of mine regarding analysis length.